media/CfgEditor/cfgeditor.html

```diff
@@ -0,0 +1,271 @@
+                        Input Path
+                        <span slot="end" class="codicon codicon-search" style="cursor: pointer"></span>
+                    </vscode-text-field>
+                </div>
+                <div class="option">
+                    <vscode-text-field id="TFLITEOutputPath" startIcon=true size=50 placeholder="">
+                        Output Path
+                        <span slot="end" class="codicon codicon-search" style="cursor: pointer"></span>
+                    </vscode-text-field>
+                </div>
+            </div>
+            <div class="basic" id="optionImportONNXBasic">
+                <div class="title">
+                    Basic Options
+                </div>
+                <div class="option">
+                    <vscode-text-field id="ONNXInputPath" startIcon=true size=50 placeholder="">
+                        Input Path
+                        <span slot="end" class="codicon codicon-search" style="cursor: pointer"></span>
+                    </vscode-text-field>
+                </div>
+                <div class="option">
+                    <vscode-text-field id="ONNXOutputPath" startIcon=true size=50 placeholder="">
+                        Output Path
+                        <span slot="end" class="codicon codicon-search" style="cursor: pointer"></span>
+                    </vscode-text-field>
+                </div>
+            </div>
+            <div class="advanced" id="optionImportONNXAdvanced">
+                <div class="title">
+                    Advanced Options
+                </div>
+                <div class="option">
+                    <vscode-checkbox id="ONNXSaveIntermediate">
+                        Save Intermediate
+                    </vscode-checkbox>
+                    <span class="codicon codicon-question" style="cursor: pointer"></span>
+                </div>
+                <div class="option">
+                    <vscode-checkbox id="ONNXUnrollRNN">
+                        Unroll RNN
+                    </vscode-checkbox>
+                    <span class="codicon codicon-question" style="cursor: pointer"></span>
+                </div>
+                <div class="option">
+                    <vscode-checkbox id="ONNXUnrollLSTM">
+                        Unroll LSTM
+                    </vscode-checkbox>
+                    <span class="codicon codicon-question" style="cursor: pointer"></span>
+                </div>
+            </div>
+        </div>
+        <div class="options" id="optionOptimize">
+            <!-- TODO Implement Optimize step -->
+        </div>
+        <div class="options" id="optionQuantize">
+            <!-- TODO Implement Quantize step -->
+        </div>
+        <div class="options" id="optionCodegen">
+            <div class="basic">
+                <div class="title">
+                    Basic Options
```

Comment:
(Q) What about removing 'Basic Options' header? I think 'Mandatory Options' is enough, so 'Basic options' can be included in it.

Reply:
Actually, 'Mandatory Options' are only in 'import' step because input model type should be selected first than any other options.
I will change "mandatory" to "premise" or "prerequisite" :)

Reply:
Actually... I think we can remove 'basic' as of now... 😃 I feel the less is the better...